action.yml

```diff
@@ -23,11 +27,17 @@ runs:
   using: 'composite'
   steps:
     - run: |
+        response=$(curl --fail-with-body --silent --show-error \
+          --request POST \
+          --header "Authorization: Bearer ${{ env.SLACK_BOT_TOKEN || inputs.token }}" \
+          --header "Content-Type: application/json; charset=utf-8" \
+          --url https://slack.com/api/chat.postMessage \
+          --data "{\"channel\": \"${{ inputs.channel }}\", \"thread_ts\": \"${{ inputs.thread_ts }}\", \"unfurl_links\": ${{ inputs.unfurl_links }}, \"text\": \"${{ inputs.text }}\"}" \
+        )
+        echo "Slack API response:\n$response"
         {
-          echo "ts=<<EOF"
-          curl --silent --show-error -X POST -H "Authorization: Bearer ${{ env.SLACK_BOT_TOKEN || inputs.token }}" -H "Content-Type: application/json; charset=utf-8" --url https://slack.com/api/chat.postMessage \
-            -d "{\"channel\": \"${{ inputs.channel }}\", \"thread_ts\":\"${{ inputs.thread_ts }}\", \"text\":\"${{ inputs.text }}\"}" \
-            | jq --raw-output '.ts'
+          echo "ts<<EOF"
+          echo "$response" | jq --raw-output '.ts'
```

Comment:
Something about how I was trying to output the `ts` output variable wasn't playing nicely with what GH was expecting:

![image](https://github.com/user-attachments/assets/be615ece-a3ac-4a43-bd51-c8687c2cdba4)

I was looking more closely at their examples for how to do multi-line output and noticed I had added the equal sign in `ts=<<EOF` that wasn't in their example: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#multiline-strings

...but I'm not sure how test test this assumption before merging this PR. 😟

Reply:
You could create a workflow in a throw-away repo that references this branch like this:

```
- name: "Post Slack message"
  id: slack-post
  uses: ynab/slack-post-message-action@db/better-logging
  with:
    channel: "#my-channel"
    text: "Test 123"
- run: echo ${{ steps.slack-post.outputs.ts }}
```

Reply:
The bigger issue here was that I needed to define how the `ts` output gets its value. The `Invalid format` aspect _did_ need to be fixed, but didn't actually correct the problem I was debugging: the `ts` output not being available in later steps for actions using this shared action.